client/src/reducers/postingsReducer.js

```diff
@@ -3,9 +3,46 @@ import ResultData from '../components/ResultData';
-    new ResultData("CPU maker", "Make CPUs and stuff. You also do things that make this string longer, so I can test how well this works for long strings. Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong ", "Make CPUs and stuff.",  "Vancouver, WA, USA", "https://ded.increpare.com/~locus/yourname.html", "https://ded.increpare.com/~locus/yourname.html"),
-    new ResultData("Software Developer", "Develop Software, but more text", "Develop Software", "Vancouver, BC, Canada", "https://github.com/mattdiamond/fuckitjs", "https://github.com/ajalt/fuckitpy")
+    new ResultData(
+      "Nothing do-er",
+      "http://hasthelargehadroncolliderdestroyedtheworldyet.com/",
+      "nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop nop",
+      "Vancouver",
+      "BC",
+      "Canada",
+      "0.000",
+      "0.000",
+      "Google",
+      "5",
+      new Date()
+    ),
+
+    new ResultData(
+      "CPU maker",
+      "https://github.com/mattdiamond/fuckitjs",
+      "Make CPUs and stuff. You also do things that make this string longer, so I can test how well this works for long strings. Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong Looooooooong ",
+      "Vancouver",
+      "BC",
+      "Canada",
+      "0.000",
+      "0.000",
+      "Microsoft",
+      "5",
+      new Date()
+    ),
+
+    new ResultData("Software Developer",
+    "https://thedailywtf.com/articles/you-can-only-get-what-you-have",
+    "Develop Software, short description",
+    "Vancouver",
+    "WA",
+    "USA",
+    "0.000",
+    "0.000",
+    "Facebook",
+    "5",
```

Comment:
I don't think redux is being used anymore in our client project. In feature/121 the redux stuff is commented out so it actually doesn't do anything. Also I'm not sure that we should have dummy data anymore since the front end is able to hit the backend. I'm not sure about that last point though because it may make front end development easier. Regardless, I don't think the dummy data should be here.

Reply:
If you are keeping t he dummy data, I think the longitude, latitude, and salaryMin should be numbers, not strings.